packages/e2e-tests/specs/performance/post-editor.test.js

```diff
@@ -97,6 +99,18 @@ function getHoverEventDurations( trace ) {
 jest.setTimeout( 1000000 );
 
 describe( 'Post Editor Performance', () => {
+	beforeAll( async () => {
+		await activateTheme( 'tt1-blocks' );
```

Comment:
This might cause some issues for us to run the perf tests against old branches. Can we add a typing measure to the site editor instead and just keep the classic theme here?

Reply:
Yep, I'll find another way. I'm just adding it for now to check the difference.

Reply:
Not sure how we'd test a large post in either the site or template editor. Guess we'll need to embed it in a post content block.

Reply:
Done. Added typing performance for the site editor.